### PR TITLE
Adding support notice

### PR DESCRIPTION
**Pull request by gauntface**
cc @jeffposnick @addyosmani @philipwalton

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -56,11 +56,20 @@ If you’ve found an error in this library, please file an issue at https://gith
...
+
+The team behind sw-toolbox and sw-precache have been busy creating [Workbox](https://workboxjs.org), which is a collection of libraries and build tools that make it easy build offline web apps. It’s a joining of [sw-toolbox](https://github.com/GoogleChrome/sw-toolbox) and [sw-precache](https://github.com/GoogleChrome/sw-precache) with more features and a modern codebase.
+
+### What does this mean for sw-toolbox?
+
+For now, it means we’ll continue to support both sw-toolbox and sw-precache with critical bug fixes and releases. However, non-critical bugs are unlikely to be addressed.
+
+### Should you switch to Workbox?
+
+We would recommend Workbox for new projects, but there is no immediate need to switch if sw-toolbox / sw-precache meets all your needs in your current project. We will announce a deprecation plan for these modules once we are happy with Workbox.
```
Comment (philipwalton) [inline README.md]: This kinda implies we're not happy with Workbox, and I fear may deter usage.

What if instead we said something like "...once Workbox has feature parity with..." or "...once Workbox fully meets the use cases of current precache/toolbox users" or something like that.

File: README.md [hunk 2]
```diff
@@ -56,11 +56,20 @@ If you’ve found an error in this library, please file an issue at https://gith
 
 Patches are encouraged, and may be submitted by forking this project and submitting a [pull request through this GitHub repo](https://github.com/GoogleChrome/sw-toolbox/pulls).
 
-## Future of Service Worker tooling
+## Support
+
+The team behind sw-toolbox and sw-precache have been busy creating [Workbox](https://workboxjs.org), which is a collection of libraries and build tools that make it easy build offline web apps. It’s a joining of [sw-toolbox](https://github.com/GoogleChrome/sw-toolbox) and [sw-precache](https://github.com/GoogleChrome/sw-precache) with more features and a modern codebase.
```
Comment (philipwalton) [inline README.md]: Nit: I suggest wording it "a collection of libraries and tools" (omit "build"), so we don't have the word "build" twice in different contexts in the same sentence.

Comment (jeffposnick) [inline README.md]: - `sw-toolbox` and `sw-precache` (Put backticks around them, here and elsewhere. I'm a little obsessive about this.)
- "easy build" -> "easy to build"

**Comment by gauntface** [comment]
cc @jeffposnick @philipwalton should have your comments addressed.

**Pull request merged**